.circleci/config.yml

```diff
@@ -108,7 +108,7 @@ references:
     run:
       name: Generate API Docs
       command: |
-        TESTS=$(circleci tests glob "spec/swagger/*_helper.rb" "spec/requests/**/*_spec.rb" | circleci tests split )
```

Comment:
We renamed the request specs but we are now no longer testing the swagger directory specs?